delete.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Cogctl.Actions.Relays.Delete do
+  use Cogctl.Action, "relays delete"
+
+  def option_spec do
+    []
+  end
+
+  def run(options, args, _config, %{token: nil}=endpoint) do
+    with_authentication(endpoint, &run(options, args, nil, &1))
+  end
+  def run(_options, [], _config, _endpoint), do: display_arguments_error
```

Comment:
ok. scratch that... I lost the ability to delete multiple relays when I specified the name in option_spec. :disappointed: With this implementation, I get a list of relay groups that I can delete.